### ISSUE TITLE
magenta.js with tensorflow.js

### ISSUE THREAD

**Issue by naotokui** [tensorflow/magenta-js#113]
I'm trying to use tensorflow.js alongside with magenta.js, to load my custom tfjs model, but it seems there is some compatibility issue. 

Here is a codepen to illustrate the issue. 
https://codepen.io/naotokui/pen/VBxvvE

I copied Tero Parviainen's pen and added 
`<script src="https://cdnjs.cloudflare.com/ajax/libs/tensorflow/0.12.4/tf.min.js">
</script>` to the html file. 

Then I hit the generate sequence button, I got this error:
`Uncaught (in promise) Error: Argument 'tensors[0]' passed to 'concat' must be a Tensor, but got object.
    at o (VM221 magentamusic.min.js:7)
    at i (VM221 magentamusic.min.js:7)
    at VM221 magentamusic.min.js:7
    at Array.forEach (<anonymous>)
    at t (VM221 magentamusic.min.js:7)
    at Object.n.assertArgumentsAreTensors (VM221 magentamusic.min.js:7)
    at n.concat (VM221 magentamusic.min.js:7)
    at VM221 magentamusic.min.js:7
    at Object.t.tidy (VM221 magentamusic.min.js:7)
    at Object.e.value [as concat] (VM221 magentamusic.min.js:7)`

If I import magenta.js before tensorflow.js, then I got this:
`magentamusic.js:1361 Uncaught Error: Unknown feature IS_BROWSER.
    at t.evaluateFeature (VM768 magentamusic.min.js:7)
    at t.get (VM768 magentamusic.min.js:7)
    at VM765 tf.min.js:2
    at VM765 tf.min.js:2
    at VM765 tf.min.js:2`
https://codepen.io/naotokui/pen/yqjYXM

**Comment by adarob** [comment]
Can you try only importing magentamusic and then using mm.tf?

**Comment by naotokui** [comment]
I got errors when I tried to use `mm.tf.sequential`, for example. 
https://codepen.io/naotokui/pen/EpLjYj

I guess that's because `mm.tf` points `tfjs-core` and `mm.tf.sequential` is in `tfjs-layers`.

**Comment by adarob** [comment]
Yes, this is a good point. We will need to package the full tfjs and not just tfjs-core for this to work. @cghawthorne @teropa any objections?

**Comment by cghawthorne** [comment]
Seems reasonable to me.
